.gitmodules

```diff
@@ -1,9 +1,9 @@
-[submodule "gapic-generator/api-common-protos"]
+[submodule "shared/api-common-protos"]
 	path = shared/api-common-protos
-	url = https://github.com/googleapis/api-common-protos.git
-[submodule "gapic-generator/protobuf"]
-	path = shared/protobuf
-	url = https://github.com/protocolbuffers/protobuf.git
-[submodule "protos/googleapis"]
+	url = git@github.com:googleapis/api-common-protos.git
+[submodule "shared/googleapis"]
 	path = shared/googleapis
-	url = https://github.com/googleapis/googleapis.git
+	url = git@github.com:googleapis/googleapis.git
+[submodule "shared/protobuf"]
+	path = shared/protobuf
+	url = git@github.com:protocolbuffers/protobuf.git
```

Comment:
Do we prefer to use git urls vs https?

Reply:
I had a hard time updating the submodules cleanly, so i removed and re-added them.

Reply:
I didn't change this on purpose, that was just how i added them back. I think the git protocol is considered faster than https, so I just always default to using it.

Reply:
👍 that works for me.